### ISSUE TITLE
Objectivos do projecto

### ISSUE THREAD

**Issue by lpalgarvio** [heldergg/dre#3]
Boas

Que objectivos futuros tens para este projecto?

Confronto de alguma espécie com o estado, para que adoptem o teu trabalho e o tornem universal e gratuito?

Procura de apoiantes entre mecenas, partidos, empresas, para fazerem pressão e criarem mudanças no DRE?

Tornar o site mainstream e começar a usar um domínio próprio?

**Comment by lpalgarvio** [comment]
This one was about the objectives of the project.

Is this about just coding something that is challenging?

Are you trying to confront the state, so they adopt your work and make it universal and free on their website (dre.pt)?

Do you seek supporters among mecenas, parties, companies, etc, to make pressure on the state and force them to make changes on DRE (like freeing access)?

Do you wish to make the site mainstream and start using a proper domain that can be more recognizable?

Cheers

**Comment by heldergg** [comment]
Eu sou uma desgraça, levei quase um ano a descobrir que tinha tickets pendentes! 

> Que objectivos futuros tens para este projecto?

Os objectivos deste projecto já foram atingidos, o objectivo principal é disponibilizar o conteúdo do dre.pt:

- Em formato apropriado para a web, html;
- Ter os documentos indexados nos motores de busca;
- Ter um motor de busca interno a funcionar.

Assim, de momento, não objectivos imediatos a cumprir.

No futuro, de acordo com o tempo que tiver e a minha vontade, gostaria de testar outras ideias e fazer vários melhoramentos. Por exemplo:

- Migrar o projecto para Python 3;
- Criar um sistema de alarmes por correio electrónico, XMPP, SMS, etc;
- Fazer uns quantos videocasts sobre a utilização do sistema (especialmente obtenção de RSS com queries associadas);
- Fazer um parsing melhor dos documento de modo a pudermos lincar, por exemplo, artigos, capítulos, etc, como o Jorge Leitão fez no publicos.pt.

> Confronto de alguma espécie com o estado, para que adoptem o teu trabalho e o tornem universal e gratuito?

Não desejo confrontos com ninguém, muito menos com o estado. De qq forma parece que [vão mudar alguma coisa até ao fim do ano](http://www.jornaldenegocios.pt/economia/detalhe/as_medidas_que_ja_se_conhecem_do_simplex_ii.html). Veremos.

> Procura de apoiantes entre mecenas, partidos, empresas, para fazerem pressão e criarem mudanças no DRE?

Não. Gostava que isto não fosse um projecto de apenas uma pessoa, estou disponível a receber PRs, patches e afins.

> Tornar o site mainstream e começar a usar um domínio próprio?

Tornar isto comercial? Não, claro que não, isso iria contra o objectivo do próprio projecto.

**Comment by lpalgarvio** [comment]
Quando dizia confronto, dizia stunt/entrevista nos media, petições, newsletters de terceiros, revistas, partidos, contacto directo com entidades governamentais, etc, tudo com intuito de promoção.

Quando referia a mainstream, não referia-me a comercial, mas sim tornar-se a fonte pública #1, semi ofcial, para este tipo de informação, e se necessário, suportada através de financiamento via crowdfunding, paypal, entidades que tenham interesse em patrocinar (como partidos, agências de advogados, etc), por forma a levar a informação ao público em geral e não só a um punhado de gente que por acaso descobriu o projecto.

**Comment by heldergg** [comment]
> Quando dizia confronto, dizia stunt/entrevista nos media, petições, newsletters de terceiros, revistas, partidos, [...]

Não, não tenho nada disso previsto. Para dizer a verdade essa vertente de um projecto não me interessa muito, ou mesmo nada. 

Além disso estamos em fase de transição, se o novo site do estado tiver qualidade suficiente, não servirá de muito manter este site.

> Quando referia a mainstream, não referia-me a comercial, mas sim tornar-se a fonte pública #1, semi ofcial, para este tipo de informação, e se necessário, suportada através de financiamento via crowdfunding, [...]

Não tenho objectivos desses. Se as pessoas utilizam e é útil, excelente, se não, também. Por um lado é verdade que a situação de alojamento pode mudar de um momento para o outro, por outro lado estou nesta situação já há uns anos, vamos ver.

Lembrei-me agora, outra funcionalidade que quero acrescentar é disponibilizar os documentos em formato ePub e/ou mobi.
